### PR TITLE
Level transitions

### LINKED ISSUE #37: Basic Level transitions
- [ ] Transition from one level to another on interacting with a particular tile

### PR DESCRIPTION
**Pull request by Exonoesis**
## What is the purpose of these changes?
Games traditionally have a world split into different sections called levels. Our game is also structured in this manner, and as such, we needed a way to transition out of one level and into another.

## What has changed?
We now have tiles that when interacted via the "E" key will transition from the first testing level (outside) to the second testing level (inside). We also fixed the movement stuttering.

## How can I verify this works?
1. Run the code and press play to enter the overworld of Level_0.
2. Use WASD to navigate to the front of the cabin and position the player directly south of the door.
3. Press the "E" key.
4. See that the level has changed to that of Level_2, the inside of the cabin.
5. Hear that the music has changed.
6. Feel free to explore this location and see that all the collision works.
7. Enter the doorway and press E again to return to Level_0.
---
Closes #37

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,13 @@
 - This project uses [ISO Standard](https://www.iso.org/iso-8601-date-and-time-format.html) date formatting
 
 ## [Unreleased]
+### Added
+- Interaction Events.
+
 ### Fixed
 - macOS Sonoma crashing the application by updating dependencies.
 - Missing File Finder thinking assets folder doesn't exist for release builds.
+- Movement no long stutters.
```
Comment (divark) [inline CHANGELOG.md:14]: It still happens unfortunately :c 

I'd recommend making a new issue to investigate the movement stuttering on its own.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 ![project image](snapshots/Snapshot_6.png)
 ![project image](snapshots/Snapshot_5.gif)
+![project image](snapshots/Snapshot_7.png)
```
Comment (divark) [inline README.md:7]: Would you mind turning this into a gif showing the player entering and leaving the cabin?

**Review by divark — CHANGES_REQUESTED** [review]
Everything seems to check out logically from my end! I just have a few cosmetic and logistical recommendations before we merge this PR.

File: CHANGELOG.md (hunk 1 above)
Comment (Exonoesis) [reply CHANGELOG.md:14]: My disappointment is immeasurable and my day is ruined. I'll remove it

**Pull request merged**